docs/whats-new.md

```diff
@@ -3,28 +3,60 @@
 In addition to these notes, always check the [Upgrade Guide](/#/documentation/upgrade-guide) when considering adopting a new release.
 
 
+
 ## Version 5.2
 
 **IN DEVELOPMENT**
 
 Date: Target April, 2018
 
-## Transform class (New, WebGL2)
+<table style="border: 0;" align="center">
+  <tbody>
+    <tr>
+      <td>
+        <img height=150 src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/webgl2.jpg" />
+        <p><i>New Transform Class</i></p>
+      </td>
+      <td>
+        <img height=150 src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/webgl2.jpg" />
+        <p><i>Example 14</i></p>
+      </td>
+      <td>
+        <img height=150 src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/webgl2.jpg" />
+        <p><i>Example 15</i></p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+## Transform class (New, WebGL2) (Experimental)
+
+The new experimental [`Transform`](/#/documentation/api-reference/transform) class provides an easy-to-use interface to perform Transform Feedback operations.
+
+
+## Framebuffer Class
+
+** "Asynchronous" Pixel Readback** (WebGL2) - A new method [`readPixelsToBuffer`](/#/documentation/api-reference/framebuffer) is added to asynchronously read pixel data into a `Buffer` object. This way applications have the freedom to reduce the CPU-GPU sync time or completely avoid it by using the `Buffer` object as data source for the GPU.
```

Comment:
Looks like ** notation not getting applied here, may be remove quotes around `Asynchronous`